nature/models.py

```diff
@@ -320,42 +297,55 @@ class Species(ProtectedNatureModel):
     code = models.CharField(max_length=20, blank=True, null=True, db_column='koodi')
```

Comment:
I was informed that the final database we will receive (an update on the one I gave you) actually has max_length=100 for the code here.

Probably it is better to make the changes now, before the models are managed, so the code will be compatible with the new database dump I will get soon?

Reply:
Good point